socorro/external/es/connection_context.py

```diff
@@ -73,19 +46,6 @@ class ConnectionContext(RequiredConfig):
         doc='the default doctype to use in elasticsearch',
         reference_value_from='resource.elasticsearch',
     )
-    required_config.add_option(
-        name='elasticsearch_connection_wrapper_class',
-        default=Connection,
-        doc='a classname for the type of wrapper for ES connections',
-        reference_value_from='resource.elasticsearch',
-    )
-
-    # Operational exceptions are retryable, conditionals require futher
-    # analysis to determine if they can be retried or not.
-    operational_exceptions = (
-        elasticsearch.exceptions.ConnectionError,
-    )
-    conditional_exceptions = ()
```

Comment:
`_submit_crash_to_elasticsearch` has its own retry logic, so we don't additionally use the `retry` function.